env0/resource_template.go

```diff
@@ -24,107 +24,34 @@ func resourceTemplate() *schema.Resource {
 	}
 }
 
-func templateCreatePayloadFromParameters(d *schema.ResourceData) (client.TemplateCreatePayload, diag.Diagnostics) {
```

Comment:
Most if it is now done automatically by readResourceData - lots of code removed now (:

Reply:
Awesome! 👏